docs/overview/multiparty/deterministic.md

```diff
@@ -121,4 +116,4 @@ Not to be overlooked is the sometimes astonishing efficiency increase that can b
 business relationships, by being able to agree the order and sequence of a set of events. Having the
 tools to digitize processes that previously took physical documents flying round the world, into
```

Comment:
Line `58` needs a link to the tokens section. 
Line `74` needs a link to the on-chain/off-chain coordination page.